misc/build_scripts/build.oasis3.ubuntu20.04

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
Don't really like the location of this file but not sure where to put it.   The misc directory looks like it is mostly old stuff that should be removed.

Wondering if we should have a directory for CI related files, maybe test/ci ?  Since it is related to ci/testing?

Fine for now but just FYI I'm creating a ticket to cleanup misc so may decide to move this file.

Reply:
> Don't really like the location of this file but not sure where to put it.

I agree. I'll move this file the next time when I update the OASIS CI workflow.